<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>471</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>471</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Estilo general del fondo */
QWidget {
    background-color: #F5F5F5; /* Fondo claro */
    color: #333333; /* Texto en gris oscuro */
    font-family: Arial, sans-serif;
    font-size: 14px;
}

/* Estilo de los botones */
QPushButton {
    background-color: #4CAF50; /* Verde oscuro */
    color: #FFFFFF; /* Texto en blanco */
    border: none;
    padding: 10px 20px;
    text-align: center;
    text-decoration: none;
    font-size: 14px;
    margin: 4px 2px;
    border-radius: 8px;
}

QPushButton:hover {
    background-color: #45a049; /* Verde más claro al pasar el ratón */
}

QPushButton:pressed {
    background-color: #388E3C; /* Verde más oscuro al presionar */
}

/* Estilo de los labels */
QLabel {
    color: #333333; /* Texto en gris oscuro */
    font-size: 14px;
}

/* Estilo del título de las ventanas */
QLabel#LoginTitle, QLabel#RegisterTitle {
    background-color: #4CAF50; /* Verde oscuro */
    color: #FFFFFF; /* Texto en blanco */
    font-size: 24px;
    font-weight: bold;
    border-radius: 8px;
    padding: 10px;
    text-align: center;
}

/* Estilo de los campos de entrada */
QLineEdit, QDateEdit, QComboBox {
    padding: 10px;
    border: 1px solid #4CAF50; /* Borde verde oscuro */
    border-radius: 4px;
    background-color: #FFFFFF; /* Fondo blanco */
    color: #333333; /* Texto en gris oscuro */
    font-size: 14px;
}

QLineEdit:focus, QDateEdit:focus, QComboBox:focus {
    border: 1px solid #45a049; /* Borde verde más claro al enfocar */
}</string>
  </property>
  <layout class="QVBoxLayout" name="ColumnaPrincipal">
   <property name="spacing">
    <number>16</number>
   </property>
   <property name="leftMargin">
    <number>16</number>
   </property>
   <property name="topMargin">
    <number>16</number>
   </property>
   <property name="rightMargin">
    <number>16</number>
   </property>
   <property name="bottomMargin">
    <number>16</number>
   </property>
   <item>
    <widget class="QLabel" name="LoginTitle">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="text">
      <string>Iniciar Sesión</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="Campos">
     <item>
      <widget class="QLabel" name="UsuarioLabell">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Email</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="EmailEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="ContrasnaLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LayoutDirection::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Conraseña</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="PasswordEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="Botones">
     <item>
      <widget class="QPushButton" name="LoginButon">
       <property name="text">
        <string>Iniciar Sesión</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="RegisterButtom">
       <property name="text">
        <string>Registrarse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
